src/Uno.UI/UI/Xaml/Window/Native/NativeWindowWrapper.Android.cs

```diff
@@ -135,12 +135,6 @@ private bool IsNavigationBarTranslucent()
 
 	private WindowInsetsCompat GetWindowInsets(Activity activity)
 	{
-		var decorView = activity.Window.DecorView;
-		if (decorView.IsAttachedToWindow)
-		{
-			return ViewCompat.GetRootWindowInsets(decorView);
-		}
-
```

Comment:
Removing this would mean we always return `null` here if we are below API 30, is that the intended behavior?

Reply:
Nope, my bad. Fixed.